clippy_lints/src/doc.rs

```diff
@@ -322,7 +322,7 @@ fn check_doc(cx: &EarlyContext, valid_idents: &[String], docs: &[(&str, Span)])
 
                         if new_line {
                             let mut lookup_parser = parser.clone();
-                            if let Some(_) = lookup_parser.find(|&(_, c)| c == ']') {
+                            if lookup_parser.any(|(_, c)| c == ']') {
```

Comment:
This lint will suggests `.find(…).is_some()`, right? Is there a lint to suggest `any` when the `find` result is unused?